### ISSUE TITLE
when i compile the dll of onednn wit debug, it failds to build, but the release is built successfully。

### ISSUE THREAD

**Issue by feixuedudiao** [oneapi-src/oneDNN#1851]
the machine is Intel(R) Core(TM) i7-10700 CPU @ 2.90GHz , win10 system, vs2022
the bat is above:
```
CALL "D:\Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvars64.bat";

mkdir "cpu_seq_cache_omp_release_x64"
cd "cpu_seq_cache_omp_release_x64"
cmake -A x64 -DBUILD_TESTS=OFF -DCMAKE_BUILD_TYPE=release -DBUILD_SHARED_LIBS=OFF -DONEDNN_BUILD_GRAPH=OFF -DONEDNN_CPU_RUNTIME=SEQ -DONEDNN_CPU_RUNTIME=OMP -DONEDNN_ENABLE_PRIMITIVE_CACHE=ON -DONEDNN_ENABLE_PRIMITIVE_CPU_ISA=ALL -DONEDNN_ENABLE_CPU_ISA_HINTS=ON -DONEDNN_VERBOSE=OFF ..

cmake --build . --config Release

cd ..

mkdir "cpu_seq_cache_omp_debug_x64"
cd "cpu_seq_cache_omp_debug_x64"

cmake -A x64 -DBUILD_TESTS=OFF -DCMAKE_BUILD_TYPE=debug -DBUILD_SHARED_LIBS=OFF -DONEDNN_BUILD_GRAPH=OFF -DONEDNN_CPU_RUNTIME=SEQ -DONEDNN_CPU_RUNTIME=OMP -DONEDNN_ENABLE_PRIMITIVE_CACHE=ON -DONEDNN_ENABLE_PRIMITIVE_CPU_ISA=ALL -DONEDNN_ENABLE_CPU_ISA_HINTS=ON -DONEDNN_VERBOSE=OFF ..

cmake --build . --config Debug
```

the error information of building oneDnn:
```
  Building Custom Rule F:/Share/OneDNN/v3.3.6/oneDNN-3.3.6_no_verbose/src/CMakeLists.txt
    ????? F:/Share/OneDNN/v3.3.6/oneDNN-3.3.6_no_verbose/cpu_seq_cache_omp_debug_x64/src/Debug/dnnl.lib ??? F:/Share/OneDNN/v3.3.6/oneDNN-3.3.6_no_verbose/cpu_seq_cache_omp_debug_x64/src/Debug/dnnl.exp
primitive_desc_iface.obj : error LNK2019: ????????? "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl dnnl::impl::rt_dims2fmt_str(enum dnnl_primitive_kind_t,struct dnnl_memory_desc const *,s truct dnnl_memory_desc const *,struct dnnl_memory_desc const *)" (?rt_dims2fmt_str@impl@dnnl@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@W4dnnl_primitive_kind_t@@PEBUdnnl_memory_desc@@11@Z),?? "public: class std::b asic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl dnnl::impl::primitive_desc_t::info_with_runtime_dims(struct dnnl_engine *,struct dnnl_memory_desc const *,struct dnnl_memory_desc const *,struct dnnl_mem ory_desc const *,struct dnnl_memory_desc const *)" (?info_with_runtime_dims@primitive_desc_t@impl@dnnl@@QEAA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEAUdnnl_engine@@PEBUdnnl_memory_desc@@111@Z) ??????? [F:\Share\O neDNN\v3.3.6\oneDNN-3.3.6_no_verbose\cpu_seq_cache_omp_debug_x64\src\dnnl.vcxproj]
primitive_desc_iface.obj : error LNK2019: ????????? "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl dnnl::impl::rt_mds2str(enum dnnl_primitive_kind_t,struct dnnl_memory_desc const *,struct  dnnl_memory_desc const *,struct dnnl_memory_desc const *,struct dnnl_memory_desc const *)" (?rt_mds2str@impl@dnnl@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@W4dnnl_primitive_kind_t@@PEBUdnnl_memory_desc@@111@Z),? ? "public: class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl dnnl::impl::primitive_desc_t::info_with_runtime_dims(struct dnnl_engine *,struct dnnl_memory_desc const *,struct dnnl_memory_desc
const *,struct dnnl_memory_desc const *,struct dnnl_memory_desc const *)" (?info_with_runtime_dims@primitive_desc_t@impl@dnnl@@QEAA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEAUdnnl_engine@@PEBUdnnl_memory_desc@@111 @Z) ??????? [F:\Share\OneDNN\v3.3.6\oneDNN-3.3.6_no_verbose\cpu_seq_cache_omp_debug_x64\src\dnnl.vcxproj]
F:\Share\OneDNN\v3.3.6\oneDNN-3.3.6_no_verbose\cpu_seq_cache_omp_debug_x64\src\Debug\dnnl.dll : fatal error LNK1120: 2 ?????????? [F:\Share\OneDNN\v3.3.6\oneDNN-3.3.6_no_verbose\cpu_seq_cache_omp_debug_x64\src\dnnl.vcxproj]
  Building Custom Rule F:/Share/OneDNN/v3.3.6/oneDNN-3.3.6_no_verbose/tests/gtests/gtest/CMakeLists.txt
```

**Comment by vpirogov** [comment]
This is a known issue with `ONEDNN_VERBOSE=OFF` and static build discussed in #1790. The fix is available in oneDNN v3.4.

**Comment by feixuedudiao** [comment]
@vpirogov thanks.
